src/views/Main/Catalog/Components/EntityTypeList.js

```diff
@@ -65,7 +67,31 @@ export class EntityTypeList extends React.Component {
   updateFn = () => {
     EntityDataModelApi.getAllEntityTypes()
       .then((entityTypes) => {
-        this.setState({ entityTypes: Utils.addKeysToArray(entityTypes) });
+        EntityDataModelApi.getAllPropertyTypes()
+        .then((propertyTypes) => {
+          const allPropNames = {};
```

Comment:
I was trying to combine as much logic as possible (for the getSuggestions function) but I could rewrite this to use only one map